app/src/main/res/values/themes.xml

```diff
@@ -70,4 +73,10 @@
         <item name="android:textAllCaps">false</item>
     </style>
 
+    <style name="Application.GliaAndroidSdkWidgetsExample.ChoiceCard.ContentText" parent="TextAppearance.MaterialComponents.Body1">
+        <item name="android:textSize">40sp</item>
+        <item name="android:textColor">@color/color_dark_cyan</item>
+        <!-- item name="fontFamily">@font/tangerine</item -->
```

Comment:
why just have this attribute uncommented?

Reply:
`app/src/main` package is "example" package - there it shows how to modify `fontFamily`

Reply:
yeah, but why not have it really set the font?

Reply:
this tangerine font is bit annoying to leave it always on - but good enough to enable it for a sec to show that it works :)

Reply:
<img width="390" alt="Screenshot 2021-12-10 at 15 41 19" src="https://user-images.githubusercontent.com/49229744/145583070-696447eb-8b5c-4e65-a15b-353e93b9a7dc.png">

its this one

Reply:
ok. randomly commented out code might be confusing to integrators. Imo a short comment would help, like "example" etc.